backend/icpy/services/chat_service.py

```
@@ -1850,6 +1962,16 @@ async def _store_message(self, message: ChatMessage):
         """Store a message to JSONL per session."""
         try:
             session_id = message.session_id or "default"
+
+            # Ensure meta sidecar exists for implicitly-created sessions.
+            # Prefer using the first user message as a title.
+            meta_name: Optional[str] = None
+            try:
+                if getattr(message, 'sender', None) == 'user' and getattr(message, 'content', None):
+                    meta_name = self._derive_default_session_name(str(message.content))
+            except Exception:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)